### PR TITLE
Android Dockerfile and example Android app with Rubolectric tests

### PR DESCRIPTION
**Pull request by gkop**

### CODE REVIEW

File: example/android/android-studio-robolectric-example/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: android
```
Comment (btaitelb) [inline example/android/android-studio-robolectric-example/.travis.yml:1]: does it make sense to still keep this file, when the rest of the project is using circleci?

**Comment by btaitelb** [comment]
+1 this looks awesome!

**Comment by nbhartiya** [comment]
+1 cool!

File: example/android/android-studio-robolectric-example/.travis.yml (hunk 1 above)
Comment (gkop) [inline example/android/android-studio-robolectric-example/.travis.yml:1]: Honestly ideally these example projects would be submodules.

Comment (btaitelb) [inline example/android/android-studio-robolectric-example/.travis.yml:1]: Yeah, or we could add this into coshx/docker-bdd-example as an example of a working cluster with a rails server and an android client.

**Pull request merged**